site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -190,11 +190,14 @@ To upgrade FusionAuth when running with `docker-compose`:
 . Start it up: `docker-compose up`.
 . Login to the administrative UI. 
 
-If there were migrations required, they'll be applied automatically to your database if the installation is link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. If the installation is in production mode, make sure you apply the migrations out of band. 
+If there were migrations required, they'll be applied automatically to your database if the installation link:/docs/v1/tech/installation-guide/fusionauth-app#runtime-modes[has a runtime mode of development]. If the installation is in production mode, make sure you apply the migrations out of band. 
```

Comment:
Sorry to Monday morning quarter back you here... 

This was mostly true prior to version 1.19.0 when we added `fusionauth-app.silent-mode` (https://fusionauth.io/docs/v1/tech/reference/configuration/) 

Now you can use silent mode for db upgrades even in production. The reason for the changes was prior to this version we didn't have a way to lock a db across multiple nodes in a way to ensure only one node attempted the ugprade. 

In version 1.19.0 we added db locking in maint. mode so we can in theory safely perform schema updates silently with more than one node. 

If you want to manage your own upgrades, out of band is still a good option.

Reply:
Awesome, opened up another PR to reflect these comments.